gpu-operator/release-notes.rst

```diff
@@ -33,6 +33,110 @@ Refer to the :ref:`GPU Operator Component Matrix` for a list of software compone
+
+  - 580.95.05 (default, recommended)
+  - 570.195.03
+  - 535.274.02
+
+* Container Device Interface (CDI) is now enabled by default when installing the GPU Operator. 
+  The ``cdi.enabled`` field in the ClusterPolicy is now set to ``true`` by default.
+  The ``cdi.default`` field is now deprecated and will be ignored.
+
+  - When ``cdi.enabled`` is ``true`` the GPU Operator now leverages CDI support in container
+    runtimes, e.g. containerd and cri-o, for injecting GPU support into workload containers.
+    This differs from prior releases where CDI support in container runtimes was not used, and
+    instead, an ``nvidia`` runtime class configured in CDI mode was used.
+  - For OpenShift users upgrading to 25.10.0, we recommend updating the ``cdi.enabled``
+    field in ClusterPolicy to ``true`` post-upgrade. This field will not automatically be
+    updated to ``true`` since the Operator Lifecycle Manager (OLM) does not mutate custom
+    resources on operator upgrades.
+
+*  When using NVIDIA vGPU with KubeVirt / OpenShift Virtualization, on GPUs that support MIG, you now have the option to select MIG-backed vGPU instances instead of time-sliced vGPU instances.
+   To select a MIG-backed vGPU profile, label the node with the name of the MIG-backed vGPU profile.
+
+* Added support for NVIDIA HGX B300 and NVIDIA HGX GB300 NVL72.
+
+* Added support for new MIG profiles with NVIDIA HGX B3000.
+
+  * Supports these profiles:
+
+    * ``1g.34gb``
+    * ``1g.34gb+me``
+    * ``1g.67gb``
+    * ``2g.67gb``
+    * ``3g.135gb``
+    * ``4g.135gb``
+    * ``7g.269gb``
+
+  * Added an ``all-balanced`` profile that creates the following GPU instances:
+
+    * ``1g.34gb`` :math:`\times` 2
+    * ``2g.67gb`` :math:`\times` 1
+    * ``3g.135gb`` :math:`\times` 1
+ 
+
+* Added support for new MIG profiles with HGX GB300 NVL72.
+
+  * Supports these profiles:
+
+    * ``1g.35gb``
+    * ``1g.35gb+me``
+    * ``1g.70gb``
+    * ``2g.70gb``
+    * ``3g.139gb``
+    * ``4g.139gb``
+    * ``7g.278gb``
+
+  * Added an ``all-balanced`` profile that creates the following GPU instances: 
+
+    * ``1g.35gb`` :math:`\times` 2
+    * ``2g.70gb`` :math:`\times` 1
+    * ``3g.139gb`` :math:`\times` 1 
+
+Improvements
+------------
+
+* The GPU Operator now configures containerd and cri-o using drop-in files by default.  
+   When installing on microk8s, you need to set the value of the RUNTIME_CONFIG_SOURCE parameter in the CLusterPolicy to ``file=/var/snap/microk8s/current/args/containerd.toml``. 
+
+* Hardened the GPU Operator container image by using a distroless as a base image.
+
+* Validator for NVIDIA GPU Operator is now included as part of the GPU Operator container image.
+  It is no longer a separate image.
+
+Fixed Issues
+------------
+
+* TBD
+
+Known Issues
+------------
+
+* When using cri-o as the container runtime, several of the GPU Operator pods may be stuck in the ``RunContainerError`` state during installation or upgrade of GPU Operator. 
```

Comment:
So, it's not just the GPU Operator. Even if we were to update the driver version (while sticking to the same operator version), users would see this issue

cc @cdesiniotis

Reply:
You are correct. What about the below edit?
```suggestion
* When using cri-o as the container runtime, several of the GPU Operator pods may be stuck in the ``RunContainerError`` state during installation of GPU Operator, upgrade of GPU Operator, or upgrade of the GPU driver daemonset. 
```